test/integration/AuthenticatedLdpHandler.test.ts

```diff
@@ -18,6 +18,7 @@ import { SimpleResponseWriter } from '../../src/ldp/http/SimpleResponseWriter';
 import { SimpleTargetExtractor } from '../../src/ldp/http/SimpleTargetExtractor';
 import streamifyArray from 'streamify-array';
 import { createResponse, MockResponse } from 'node-mocks-http';
+import * as url from 'url';
```

Comment:
`{ URL }` ?

Reply:
True. Was originally added for the `format` function (and I didn't like just having `format` there without the url), but for `URL` it's uglier.